### PR TITLE
Remove pagemacro inclusion of image guide in HTML>Element>img

### PR DESCRIPTION
**Pull request by hamishwillee**
This removes the pagemacro inclusion of the Image guide and reference into HTML > Element > img topic on "supported image types". It further removes tidies up the image guide links to fix up some notes and links. This is part of #3196 

The change essentially just creates a smaller list and link to the relevant information.

It is also precursor to work I am going to do for #7744

### CODE REVIEW

File: files/en-us/web/html/element/img/index.html [hunk 1]
```diff
@@ -44,19 +44,26 @@
...
+	<li><a href="/en-US/docs/Web/Media/Formats/Image_types#jpeg">JPEG (Joint Photographic Expert Group image)</a> — Good choice for lossy compression of still images (currently the most popular).</li>
+	<li><a href="/en-US/docs/Web/Media/Formats/Image_types#png">PNG (Portable Network Graphics)</a> — Good choice for lossy compression of still images (slightly better quality than JPEG).</li>
+	<li><a href="/en-US/docs/Web/Media/Formats/Image_types#svg">SVG (Scalable Vector Graphics)</a> — Vector image format. Use for images that must be drawn accurately at different sizes.</li>
+	<li><a href="/en-US/docs/Web/Media/Formats/Image_types#webp">WebP (Web Picture format)</a> — Excellent choice for both images and animated images</li>
+</ul>
+
+
+<p>The newer formats like <a href="/en-US/docs/Web/Media/Formats/Image_types#webp">WebP</a> and <a href="/en-US/docs/Web/Media/Formats/Image_types#avif">AVIF</a> are highly recommended as they are now broadly (and somewhat deeply) supported by web browsers, and perform much better than PNG, JPEG, GIF for both still and animated images. SVG remains the recommended format for images that that must be drawn accurately at different sizes.</p>
```
Comment (hamishwillee) [inline files/en-us/web/html/element/img/index.html:66]: This was my only concern about the edit. IMO webp is now the format of choice unless you need to use SVG - it has broad support, and in terms of market share is "more than deep enough". That is not quite so true of AVIF. Anyway, decided it was time to start pushing the newer formats.

Comment (teoli2003) [inline files/en-us/web/html/element/img/index.html:66]:
```suggestion
<p>Formats like <a href="/en-US/docs/Web/Media/Formats/Image_types#webp">WebP</a> and <a href="/en-US/docs/Web/Media/Formats/Image_types#avif">AVIF</a> are highly recommended as they are now broadly (and somewhat deeply) supported by web browsers, and perform much better than PNG, JPEG, GIF for both still and animated images. SVG remains the recommended format for images that must be drawn accurately at different sizes.</p>
```
One typo, and de-emphasize that they are new (which is relative as WebP dates back to 2010…)

**Review by teoli2003 — APPROVED** [review]
One nit. But approving already.

File: files/en-us/web/html/element/img/index.html (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/web/html/element/img/index.html:66]: Thanks @teoli2003 !
It's a fair point. The only reason I stuck with "newer" is that FF92 is the first version with AVIF support by default. But I'm happy with this.

**Pull request merged**